README.md

```diff
@@ -135,146 +133,73 @@ Input file `users.json`
   }
 ]
 ```
- 
-Let's query users old enough to drink alcohol:
-
-`dw -i payload <fullpathToUsers.json> "output application/json --- payload filter (item) -> item.age > 17"`
 
+Let's query users old enough to drink alcohol:
 
-### Query content from standard input
+```bash
+dw -i payload <fullpathToUsers.json> "output application/json --- payload filter (item) -> item.age > 17"
+```
 
-`cat <fullpathToUser.json> | dw "output application/json --- payload filter (item) -> item.age > 17"`
+#### Output
 
+```json
+[
+  {
+    "name": "User1",
+    "age": 19
+  },
+  {
+    "name": "User2",
+    "age": 18
+  }
+]
+```
 
-### Redirecting the output to a file
+### Query Content From Standard Input
 
-`dw "output application/xml --- users: {( 1 to 100 map (item) -> {user: "User" ++ item} )}" >> out.xml` 
+```bash
+cat <fullpathToUser.json> | dw "output application/json --- payload filter (item) -> item.age > 17"
+```
 
 
-### DW Important ENV variables
+### Redirecting the Output to a File
 
-* *DW_HOME* = The directory where the home will be found if not defined `~/.dw` will be used
-* *DW_LIB_PATH* = The directory where libraries are going to be search by default. If not defined `${DW_HOME}/libs` will be used
-* *DW_DEFAULT_INPUT_MIMETYPE* = The default mimeType that is going to be used for the standard input. If not defined `application/json` will be used
-* *DW_DEFAULT_OUTPUT_MIMETYPE* = The default output mimeType that is going to be if not defined. If not defined `application/json` will be used
+```bash 
+dw "output application/xml --- users: {( 1 to 100 map (item) -> {user: "User" ++ item} )}" >> out.xml
+```
 
+### DataWeave CLI Environment Variables
```

Comment:
Shouldn't we list this in a different section? Maybe under the help example? The other sections here look like a series of use cases.